nifi-nar-bundles/nifi-standard-services/nifi-dbcp-service-bundle/nifi-dbcp-service/src/main/java/org/apache/nifi/dbcp/DBCPConnectionPool.java

```diff
@@ -221,7 +326,13 @@ public void onConfigured(final ConfigurationContext context) throws Initializati
         final String passw = context.getProperty(DB_PASSWORD).evaluateAttributeExpressions().getValue();
         final Integer maxTotal = context.getProperty(MAX_TOTAL_CONNECTIONS).asInteger();
         final String validationQuery = context.getProperty(VALIDATION_QUERY).evaluateAttributeExpressions().getValue();
-        final Long maxWaitMillis = "-1".equals(context.getProperty(MAX_WAIT_TIME).getValue()) ? -1 : context.getProperty(MAX_WAIT_TIME).asTimePeriod(TimeUnit.MILLISECONDS);
+        final Long maxWaitMillis = extractMillisWithInfinite(context.getProperty(MAX_WAIT_TIME));
+        final Integer minIdle = context.getProperty(MIN_IDLE).asInteger();
```

Comment:
Related to #3937 , these new properties should have EL evaluated on them as the doc suggests. I wrote https://issues.apache.org/jira/browse/NIFI-6954 to add the calls to `evaluateAttributeExpressions()`